### ISSUE TITLE
node express basic challenge not working on glitch

### ISSUE THREAD

**Issue by kanuos** [freeCodeCamp/freeCodeCamp#36178]
**Describe your problem and how to reproduce it:**
cloning the repository on glitch shows error "This document has been deleted outside of the editor."
myapp.js is not available too

Please run "refresh" in the console.

**Add a Link to the page with the problem:**
github clone : https://github.com/freeCodeCamp/boilerplate-express/
glitch : https://glitch.com/edit/#!/careful-november

**Tell us about your browser and operating system:**
* Browser Name: google chrome
* Browser Version:  74
* Operating System: windows 8.1

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

**Comment by RandellDawson** [comment]
@kanuos You did not provide the url of the actual challenge.

**Comment by minggas** [comment]
@kanuos I think that glitch link that you provided is incorrect. The actual link to match the github's repo is https://learn.freecodecamp.org/apis-and-microservices/basic-node-and-express/

**Comment by thepeted** [comment]
*Add a Link to the page with the problem:*
https://learn.freecodecamp.org/apis-and-microservices/basic-node-and-express

This page references a broken link which is intended to clone the exercise repo in to a new glitch project: https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-express/ 

As an experiment, I tried to import the repo via Glitch UI (New Project > Clone From Git Repo). This worked however was not usable because the exercise files are not kept in branch `master` but instead in `gomix`.

By manually creating the files in a new project, I could successfully verify the first two exercises.

**Comment by huyenltnguyen** [comment]
I think this is an issue with Glitch.

It was reported in #34798, and in the forum as well (https://www.freecodecamp.org/forum/t/help-with-glitch-introduction-to-the-basic-node-and-express-challenges/251566).

TL;DR: You need to make sure you're logged in with your GitHub account in order for it to work.

---
I'm closing this issue as it is a duplicate. Further discussion should be in posted the other issue thread.

Happy coding!

**Comment by Vibha27** [comment]
It is not submitting after adding link
